components/applications-service/pkg/storage/postgres/schema/sql/03_service_group_view.up.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE service ADD COLUMN full_pkg_ident TEXT NOT NULL DEFAULT '';
+
+UPDATE service SET full_pkg_ident =
+  CONCAT (service.origin, '/', service.name, '/', service.version, '/', service.release);
+
+CREATE OR REPLACE VIEW service_group_health AS
```

Comment:
Thank you so much for changing the name of the view! 💯 -- Could you please also rename this file to `03_service_group_health_view.up.sql` to match the new name? 🥇